meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/imx8mp-lpddr4-evk-sec/full_image.uuu.in

```diff
@@ -0,0 +1,30 @@
+uuu_version 1.2.39
+
+SDPS: boot -f imx-boot-mfgtool.signed
+
+SDPV: delay 1000
+SDPV: write -f u-boot-mfgtool.itb
+SDPV: jump
+
+FB: ucmd setenv fastboot_dev mmc
+FB: ucmd setenv mmcdev ${emmc_dev}
+FB: ucmd mmc dev ${emmc_dev} 1; mmc erase 0 0x2000
+
+# Clear fiovb vars
+FB: ucmd fiovb init ${mmc_dev}
```

Comment:
Could you please use the universal block of "Clear fiovb vars" for both closed/open boards, similar to `meta-lmp-bsp/recipes-support/mfgtool-files/mfgtool-files/imx8mm-lpddr4-evk-sec/full_image.uuu.in`?

Reply:
so I believe the fundamental assumption of this script is that the board is already fused and closed. If that is the case, then you should fail this script instead of ignoring all the variables and flashing the rest of the secure spl, u-boot and wic

Reply:
@MrCry0 done

Reply:
@Tim-Anderson correct, we assume the board is fused and closed, are you suggesting to change the script for all -sec machines to avoid flashing the secure artifacts if imx_is_closed is false?